setup-build.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import os
+import platform
+
+dist = platform.dist()[0]
+common_tools = "cmake flex bison"
```

Comment:
We provide grammar.c to avoid the need for flex and bison

Reply:
I will remove it.